File: wtplayer/src/WTShadow.cpp
#include "basetypes.hpp"
#include "WTShadow.hpp"

int WTShadow::getType(){
  APILOG;
  return 0;
}

void WTShadow::setEnable(bool enabled){
  APILOG;
}

bool WTShadow::getEnable(){
  APILOG;
  return false;
}

void WTShadow::setFieldOfView(float Fov){
  APILOG;
}

float WTShadow::getFieldOfView(){
  APILOG;
  return 0;
}

void WTShadow::setResolution(int Width,
                   int Height){
  APILOG;
}

int WTShadow::getWidth(){
  APILOG;
  return 0;
}

int WTShadow::getHeight(){
  APILOG;
  return 0;
}

void WTShadow::addCaster(WTGroup* Group_To_Add,
               int Red,
               int Green,
               int Blue){
  APILOG;
}

void WTShadow::removeCaster(WTGroup* Group_To_Remove){
  APILOG;
}

void WTShadow::addReceiver(WTGroup* Group_To_Add){
  APILOG;
}

void WTShadow::removeReceiver(WTGroup* Group_To_Remove){
  APILOG;
}

void WTShadow::setCasterColor(WTGroup* Group,
                    int Red,
                    int Green,
                    int Blue){
  APILOG;
}

int WTShadow::getCasterColorRed(WTGroup* Group){
  APILOG;
  return 0;
}

int WTShadow::getCasterColorGreen(WTGroup* Group){
  APILOG;
  return 0;
}

int WTShadow::getCasterColorBlue(WTGroup* Group){
  APILOG;
  return 0;
}

void WTShadow::setFiltering(int FilteringMethod){
  APILOG;
}

int WTShadow::getFiltering(){
  APILOG;
  return 0;
}

void WTShadow::setRange(bool RelativeToCaster,
              float Near,
              float Far){
  APILOG;
}

void WTShadow::enableCulling(bool Enable,
                   bool EnableExtraCullPlanes){
  APILOG;
}

File: wtplayer/javasrc/wildtangent/webdriver/WTShadow.java
package wildtangent.webdriver;

import wildtangent.webdriver.WTGroup;

public interface WTShadow
extends WTGroup {
    public void setRange(boolean var1, float var2, float var3);

    public void setRange(boolean var1, float var2);

    public void setRange(boolean var1);

    public int getCasterColorRed(WTGroup var1);

    public void enableCulling(boolean var1, boolean var2);

    public void setResolution(int var1, int var2);

    public int getCasterColorBlue(WTGroup var1);

    public void setCasterColor(WTGroup var1, int var2, int var3, int var4);

    public void setFieldOfView(float var1);

    public float getFieldOfView();

    public void setFiltering(int var1);

    public int getFiltering();

    public void setEnable(boolean var1);

    public boolean getEnable();

    public void removeCaster(WTGroup var1);

    public void removeReceiver(WTGroup var1);

    public int getType();

    public int getHeight();

    public void addCaster(WTGroup var1, int var2, int var3, int var4);

    public void addCaster(WTGroup var1, int var2, int var3);

    public void addCaster(WTGroup var1, int var2);

    public void addCaster(WTGroup var1);

    public void addReceiver(WTGroup var1);

    public int getCasterColorGreen(WTGroup var1);

    public int getWidth();
}

